flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-arches": [
+        "arm"
```

Comment:
you can remove this file, arm hasn't been supported for a couple of years already.

Reply:
Might want to review the "flathub.json" paragraph on [this page](https://github.com/flathub/flathub/wiki/App-Maintenance) then as the only reason I added the file was that specific paragraph which explicitly mentions an arm build.